content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,19 +2,19 @@
 Multiple sales teams
 ====================
 
-Odoo lets you manage several sales teams, departments, or channels, each with their own unique
-sales processes, using *Sales Teams*.
+Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
+their own unique sales processes.
 
 
 Create a new Sales Team
 =======================
 
-To create a new Sales Team, go to :menuselection:`CRM --> Sales --> Teams` then click
+To create a new Sales Team, go to :menuselection:`CRM --> Configuration --> Sales Teams` then click
 :guilabel:`Create`.
```

Comment:
`Sales Team` doesn't need to be capitalized here, as it was already introduced as a term in the overview and it's not a proper noun.
```suggestion
To create a new sales team in Odoo, go to :menuselection:`CRM --> Configuration --> Sales Teams`
then click :guilabel:`Create`.
```